src/Player.java

```diff
@@ -1,21 +1,26 @@
 public class Player {
-    private String name;
-    private int score;
 
-    public Player(String name) {
-        this.name = name;
-    }
+  private String name;
+  private int score;
 
-    public void incrementScore(int n) {
-        score += n;
-    }
+  public Player(String name) {
+    this.name = name;
+  }
 
-    public void decrementScore(int n) {
-        score -= n;
-    }
+  public void incrementScore(int n) {
+    score += n;
+  }
 
-    @Override
-    public String toString() {
-        return name;
-    }
+  public void decrementScore(int n) {
+    score -= n;
+  }
+
+  public Integer getScore(){
```

Comment:
You wanna return the non-boxed `int` type here. Integer only really appears as an argument to a generic type parameter.